versioned_docs/version-2.0.0/quickstart/sample-rust-crud-mongo.md

```diff
@@ -0,0 +1,146 @@
+---
+id: sample-rust-crud-mongo
+title: Rust CRUD REST API with MongoDB
+sidebar_label: Rust + MongoDB (REST)
+description: A sample CRUD REST API application to test Keploy integration capabilities using Rust and MongoDB.
+tags:
+  - Rust
+  - MongoDB
+  - REST API
+keyword:
+  - Rust
+  - MongoDB
+  - API Test generator
+  - Auto Testcase generation
+  - CRUD
+  - REST
+---
+
+## Introduction
+
+This is a sample CRUD REST API application to showcase Keploy integration capabilities using Rust and MongoDB. Let's get started! 🚀
+
+import InstallationGuide from '../concepts/installation.md'
+
+<InstallationGuide/>
+
+## Get Started! 🎬
+
+### Setup app
+
+Now that we have Keploy installed, let's set up our application.
+
+```bash
+git clone https://github.com/keploy/samples-rust
+cd samples-rust/CRUD-rust-mongo
+```
+
+## Running App Locally on Linux/WSL 🐧
+
+We will be using Docker compose to run Mongo on Docker container.
+
+### Let's start the MongoDB Instance
+
+Open the root directory path in your terminal and then execute the following command:
+
+```bash
+docker-compose up -d
+```
+
+<!-- ![TestRun](images/image1.png) -->
+
+```bash
+root@Linus:~/samples-rust/CRUD-rust-mongo# docker-compose up -d
+[+] Running 1/2
+ ⠦ Network crud-rust-mongo_default  Cr...                                   0.6s
+ ✔ Container mongo                  Started                                 0.6s
+root@Linus:~/samples-rust/CRUD-rust-mongo#
+```
+
```

Comment:
I don't think we put these logs in our docs here is reference https://keploy.io/docs/server/sdk-installation/go/